src/TestSuite/StringCompareTrait.php

```diff
@@ -43,6 +54,15 @@ public function assertSameAsFile($path, $result)
     {
         $path = $this->_compareBasePath . $path;
```

Comment:
For *nix based OS you could check for a leading slash.
``/absolute/path/`` interpret as such.
``relative/path/``, ``./another/relative/path``
But that wouldn't work for Windows, thus I asked about Windows :smirk_cat: